jme3-terrain/src/main/resources/Common/MatDefs/Terrain/AdvancedPBRTerrain.frag

```diff
@@ -2,83 +2,73 @@
 #import "Common/ShaderLib/GLSLCompat.glsllib"
 #import "Common/ShaderLib/PBR.glsllib"
 #import "Common/ShaderLib/Lighting.glsllib"
-#import "Common/MatDefs/Terrain/AfflictionLib.glsllib"
-
+#import "MatDefs/ShaderLib/AfflictionLib.glsllib"
```

Comment:
`#import "MatDefs/ShaderLib/AfflictionLib.glsllib"`

Can you please recheck this path?

Looks like the library is in a different path:

https://github.com/jMonkeyEngine/jmonkeyengine/tree/a04eba4c413c1e1417a204b693d53b340441e701/jme3-terrain/src/main/resources/Common/MatDefs/Terrain